components/storage-fabric/appcluster/storage-fabric.rook.29.deployment.operator.yaml

```
@@ -8,14 +8,13 @@
 #
 # Also see other operator sample files for variations of operator.yaml:
 # - operator-openshift.yaml: Common settings for running in OpenShift
-# - operator-with-csi.yaml: Enable the ceph-csi driver
 #################################################################################################################
-
+# OLM: BEGIN OPERATOR DEPLOYMENT
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: rook-ceph-operator
-  namespace: __NPH_NAMESPACE
+  namespace: rook-ceph
```

The previous namespace is the correct one as the compilation process will change it

```suggestion
  namespace: __NPH_NAMESPACE
```